mm2src/mm2_main/tests/mm2_tests/tendermint_tests.rs

```diff
@@ -38,7 +38,7 @@ fn test_tendermint_activation_and_balance() {
 
     let result: RpcV2Response<TendermintActivationResult> = serde_json::from_value(activation_result).unwrap();
     assert_eq!(result.result.address, expected_address);
-    let expected_balance: BigDecimal = "0.575457".parse().unwrap();
+    let expected_balance: BigDecimal = "0.572203".parse().unwrap();
```

Comment:
I wonder how this address end up with different balance. :thinking:

Reply:
> I wonder how this address end up with different balance. 🤔

Perhaps I once accidentally copied the wrong address from the error log
![image](https://github.com/KomodoPlatform/komodo-defi-framework/assets/52405288/84e8d2f4-7353-40e0-a12a-94db3101789c)

from time to time I were requesting test coins for cosmos and iris